pkg/metrics/engine.go

```diff
@@ -121,6 +122,13 @@ func newEngineMetrics() engineMetrics {
 			Name:      "payload_size",
 			Help:      "Accumulated size of all objects in a shard",
 		}, []string{shardIDLabelKey})
+
+		capacitySize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: storageNodeNameSpace,
+			Subsystem: engineSubsystem,
+			Name:      "capacity",
+			Help:      "Contains the shard's capacity, GB",
```

Comment:
And btw, I doubt we're using GB as the base unit for size in metrics. It should follow other values like `payload_size`.